python/ray/_private/services.py

```diff
@@ -787,7 +800,8 @@ def start_redis(node_ip_address,
                 redis_max_clients=None,
                 redirect_worker_output=False,
                 password=None,
-                fate_share=None):
+                fate_share=None,
+                ray_params=None):
```

Comment:
Instead of passing in the full `RayParams` object, could we just pass in the port blacklist list?